microsetta_interface/templates/source.jinja2

```diff
@@ -81,6 +87,12 @@
             return window.confirm(confirmMsg);
         }
 
+        function verifyDeleteSource(sourceId){
+            let confirmMsg = "Deleting this source will discard all surveys associated with it. " +
```

Comment:
We humans are just awful with deletion warnings--we tend to ignore them until after we hit confirm <smh>.  We might get slightly fewer mistaken deletions if we passed in the user-readable `source_name` and modified the first sentence of the message to something like 

> You are deleting source "`source_name`" and every survey associated with it!

Reply:
Done